### ISSUE TITLE
boards->info always return false

### ISSUE THREAD

**Issue by rabintang** [seregazhuk/php-pinterest-bot#278]
i call $bot->boards->info(username, boardname) using correct params, however it always returns false.

**Comment by seregazhuk** [comment]
@rabintang , can you give a code snippet? I have just checked and it works:
```php
print_r($bot->boards->info('familyfinder', 'national-parks-in-the-usa'));
```

**Comment by rabintang** [comment]
$info = $bot->boards->info("rabintang", "Oakley Sunglasses");

**Comment by seregazhuk** [comment]
@rabintang fixed in version 5.3.8, please update

**Comment by rabintang** [comment]
@seregazhuk yes, it works when i update. however, i maybe has a bug:
protected function getProvider($provider) in ProvidersContainer.php should be public?

**Comment by seregazhuk** [comment]
@rabintang oh, sorry about that. Now fixed in 5.3.9
